README.md

```diff
@@ -22,81 +22,19 @@ subxt metadata -f bytes > metadata.scale
 a different node then the `metadata` command accepts a `--url` argument.
 
-### Generating the runtime API from the downloaded metadata
+## Subxt Documentation
 
-Declare a module and decorate it with the `subxt` attribute which points at the downloaded metadata for the
-target runtime:
-
-```rust
-#[subxt::subxt(runtime_metadata_path = "metadata.scale")]
-pub mod node_runtime { }
-```
-
-**Important:** `runtime_metadata_path` resolves to a path relative to the directory where your crate's `Cargo.toml`
-resides ([`CARGO_MANIFEST_DIR`](https://doc.rust-lang.org/cargo/reference/environment-variables.html)), *not* relative to the source file.
-
-### Initializing the API client
-
-```rust
-use subxt::{ClientBuilder, DefaultConfig, SubstrateExtrinsicParams};
-
-let api = ClientBuilder::new()
-    .set_url("wss://rpc.polkadot.io:443")
-    .build()
-    .await?
-    .to_runtime_api::<node_runtime::RuntimeApi<DefaultConfig, SubstrateExtrinsicParams<DefaultConfig>>>();
-```
-
-The `RuntimeApi` type is generated by the `subxt` macro from the supplied metadata. This can be parameterized with user
-supplied implementations for the `Config` and `Extra` types, if the default implementations differ from the target
-chain.
-
-### Querying Storage
-
-Call the generated `RuntimeApi::storage()` method, followed by the `pallet_name()` and then the `storage_item_name()`.
-
-So in order to query `Balances::TotalIssuance`:
-
-```rust
-let total_issuance = api
-    .storage()
-    .balances()
-    .total_issuance(None)
-    .await
-    .unwrap()
-```
-
-### Submitting Extrinsics
-
-Submit an extrinsic, returning success once the transaction is validated and accepted into the pool:
-
-```rust
-use sp_keyring::AccountKeyring;
-use subxt::PairSigner;
-
-let signer = PairSigner::new(AccountKeyring::Alice.pair());
-let dest = AccountKeyring::Bob.to_account_id().into();
-
-let tx_hash = api
-    .tx()
-    .balances()
-    .transfer(dest, 10_000)
-    .sign_and_submit(&signer)
-    .await?;
-```
-
-For more advanced usage, which can wait for block inclusion and return any events triggered by the extrinsic, see the
-[balance transfer example](./examples/examples/balance_transfer.rs).
+For more details regarding utilizing subxt, please visit the [documentation](LIB_DOC.md).
 
 ## Integration Testing
 
 Most tests require a running substrate node to communicate with. This is done by spawning an instance of the
-substrate node per test. It requires an executable binary `substrate` at [`polkadot-v0.9.10`](https://github.com/paritytech/substrate/releases/tag/polkadot-v0.9.10) on your path.
+substrate node per test. It requires an executable binary `substrate` at the latest `polkadot` version on your path.
 
 This can be installed from source via cargo:
 
 ```bash
-cargo install --git https://github.com/paritytech/substrate node-cli --tag=polkadot-v0.9.10 --force
+cargo install --git https://github.com/paritytech/substrate node-cli --tag=polkadot-v0.9.18 --force
```

Comment:
Maybe just go for master?

For the examples, we expect the user to run a `polkadot` binary that is whatever we've checked the examples against (but latest should work). For the integration tests, we test against a nightly substrate build I think (assuming CI is working properly)